### PR TITLE
Update to make it compatible with swift 1.2

### PR DESCRIPTION
**Pull request by ericreber**

### CODE REVIEW

**Comment by robb** [comment]
Thanks :sparkling_heart:

File: Hamburger Button/AppDelegate.swift [hunk 1]
```diff
@@ -12,14 +12,17 @@ import UIKit
 class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
-    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: NSDictionary?) -> Bool {
+    
+    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
         self.window = UIWindow(frame: UIScreen.mainScreen().bounds)
-
+
```
Comment (robb) [inline Hamburger Button/AppDelegate.swift]: Could you remove the trailing whitespace?

File: Hamburger Button/AppDelegate.swift [hunk 2]
```diff
@@ -12,14 +12,17 @@ import UIKit
...
+    
+    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
         self.window = UIWindow(frame: UIScreen.mainScreen().bounds)
-
+        
         self.window!.backgroundColor = UIColor.whiteColor()
         self.window!.rootViewController = ViewController()
         self.window!.makeKeyAndVisible()
-
+
```
Comment (robb) [inline Hamburger Button/AppDelegate.swift]: same as above

Comment (ericreber) [inline Hamburger Button/AppDelegate.swift]: Done

2015-04-28 14:21 GMT+02:00 Robert Böhnke <notifications@github.com>:

**Comment by robb** [comment]
:bow:

**Pull request merged**